src/main/java/com/glencoesoftware/omero/ms/core/OmeroWebJDBCSessionStore.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (C) 2017 Glencoe Software, Inc. All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package com.glencoesoftware.omero.ms.core;
+
+import java.lang.RuntimeException;
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.CompletableFuture;
+import java.util.List;
+import java.util.Base64;
+import java.util.Properties;
+
+import org.python.core.Py;
+import org.python.core.PyString;
+import org.python.core.PyList;
+import org.python.core.PyDictionary;
+import org.python.core.util.StringUtil;
+import org.python.modules.cPickle;
+
+import java.sql.*;
+
+import org.slf4j.LoggerFactory;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.Handler;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.jdbc.JDBCClient;
+import io.vertx.ext.sql.SQLConnection;
+import io.vertx.ext.sql.ResultSet;
+
+
+/**
+ * An OMERO.web session store.
+ * @author Chris Allan <callan@glencoesoftware.com>
+ *
+ */
+public class OmeroWebJDBCSessionStore implements OmeroWebSessionStore{
+
+    private static final String SELECT_SESSION_SQL =
+        "select session_data from django_session where session_key = ?";
+
+    private static final org.slf4j.Logger log =
+            LoggerFactory.getLogger(OmeroWebJDBCSessionStore.class);
+
+    private JDBCClient client;
+    private Connection sync_connection;
+    public OmeroWebJDBCSessionStore(String url,
+        String user,
+        String password,
+        Vertx vertx) {
+        client = JDBCClient.createShared(vertx, new JsonObject()
```

Comment:
I don't really have strong feelings either way. I would lean towards not using JsonObject just because 
1.  There are only the three strings to pass in and
2. Calling the method without the required parameters is a compile-time error like this, but with JsonObject we won't know if the fields are missing until runtime.

Reply:
We'll have to load them from configuration though anyway so wouldn't that mean we'd just be calling `getString()` etc. on a JsonObject further up the call stack anyway?